src/Data/Abstract/ScopeGraph.hs

```diff
@@ -103,8 +103,8 @@ data Info scopeAddress = Info
   } deriving (Eq, Show, Ord, Generic, NFData)
 
 instance HasSpan (Info scopeAddress) where
-  span = lens infoSpan (\i s -> i { infoSpan = s })
-  {-# INLINE span #-}
+  span_ = lens infoSpan (\i s -> i { infoSpan = s })
+  {-# INLINE span_ #-}
```

Comment:
We renamed the `Span` fields to `start` and `end`, and now follow the convention that lens names end in `_`.